### DISCUSSION TITLE
Add private key to Rainbow Wallet via QR Code

### DISCUSSION THREAD

**Discussion by ackvf** [rainbow-me/rainbowkit#1632] [Feedback]
Hello, I am new user of Rainbow Wallet and immediatelly I am frustrated that I have to type the seed phrase by hand. Can you add option to import a wallet via QR code instead only by Recovery Phase or Private Key, really?

**Comment by 0xmkh** [comment]
@ackvf Thanks for the feature request! Our team is looking into it now and we will keep you updated once things are done on our end 🙏
